pcsd/config.rb

```diff
@@ -2,6 +2,12 @@
 
 require 'cluster.rb'
 require 'permissions.rb'
+require 'i18n'
+
+I18n.load_path << Dir[File.expand_path(File.dirname(__FILE__)) + "/locales/*.yml"]
+I18n.enforce_available_locales = true
+I18n.config.available_locales = [:'zh-CN', :'en-US']
+I18n.default_locale = :'en-US' # (note that `en` is already the default!)
```

Comment:
This is not the best place (file is used for config file handling not for actual configuration) for the i18n setup code. This should go either to `rserver.rb` or `bootstrap.rb`.

Also I don't see any way for users to change locale other than making changes in the code. I would expect that there will a configuration option in pcsd config file `pcsd.conf` or pcsd will accept some system wide locale settings.